### DISCUSSION TITLE
第3节课作业(22班) 

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#236] [作业]
在这里提交链接

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135532829

**Comment by lyyilin** [comment]
打卡，加油，第三天。https://blog.csdn.net/qq_73668945/article/details/135537921

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture3-a4e5d92914004f70bd2bbcd81092bbef

**Comment by miku8miku** [comment]
https://note.youdao.com/s/XQ1HGR80

**Comment by hyanworkspace** [comment]
https://github.com/hyanworkspace/InternLM/blob/main/notes/lecture3.md

**Comment by aqizhoua** [comment]
https://aqizhoua.github.io/poster/8e7b9ade.html

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by Lynn401** [comment]
https://daisy-401.notion.site/Tutorial-3-4f01d2a0c90f45f58c00bfd588749bf0?pvs=4

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_3/homework_3.md

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/homework/homework_3.md

**Comment by TAICHIKF** [comment]
http://t.csdnimg.cn/GfSYg

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/tree/Assignment

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture3.md

**Comment by leonfrank** [comment]
https://github.com/leonfrank/InternLM-HW/blob/main/lecture3/hw3.pdf

**Comment by vjsmmk** [comment]
https://github.com/vjsmmk/puyu/blob/main/3/hw/homework.pdf

**Comment by Juanting-Xu** [comment]
https://github.com/Juanting-Xu/InternLM_notes/blob/main/hw3.md

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/f3b1c4eb-ebef-485f-be98-dd763608f563

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/assignment/#l3-internlm-langchain

**Comment by cyq2368** [comment]
https://mwsnr0a94gx.feishu.cn/docx/NUzAdHquIoApslxI1mOcLSiunQh

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/ghxwxegc8fhggnz7?singleDoc#
